.github/workflows/kvrocks.yaml

```
@@ -212,11 +216,15 @@ jobs:
           echo "NPROC=$(sysctl -n hw.ncpu)" >> $GITHUB_ENV
           echo "CMAKE_EXTRA_DEFS=-DOPENSSL_ROOT_DIR=/usr/local/opt/openssl" >> $GITHUB_ENV
       - name: Setup Linux
-        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        if: ${{ startsWith(matrix.os, 'ubuntu') || matrix.arm_linux }}
```

```suggestion
        if: ${{ startsWith(matrix.os, 'ubuntu') }}
```

"Ubuntu Arm64" falls into `startsWith(matrix.os, 'ubuntu') == true`?

---

@PragmaTwice

---

ahh you are right. missed it.